src/Material/Tooltip.elm

```diff
@@ -0,0 +1,523 @@
+module Material.Tooltip
+  exposing
+    ( Model
+    , defaultModel
+    , Msg(..)
+    , update
+    , view
+    , render
+    , DOMState
+    , left
+    , right
+    , top
+    , bottom
+    , default

```

Comment:
I don't think we need a `default` property: you specify it by not doing anything else. None of the other components have it.